app/models/notice.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Notice < ApplicationRecord
+  belongs_to :user
+  validates :notice_body, :user_id, :created_at, :notice_title, presence: true
```

Comment:
Do we need to validate `created_at` separately?

Reply:
do we need to validate `user_id` separately? Thank You

Reply:
No bhai, it is fixed now.